### DISCUSSION TITLE
Improving german README

### DISCUSSION THREAD

**Discussion by bitwerkstatt** [Textualize/rich#2843] [Ideas] [answered]
To me as a german reader the README does not read like a technical text but more like a marketing document. That might be that it has been translated very literally which means that it also translated idioms and wordings that are normally not used in german at all or at least not in a technical document. American english and german differ a lot here, with german often being much more factual and less emotional. One example: The original has this line:

To effortlessly add rich output to your application, you can import...

which becomes

Um deiner Anwendung mühelos eine Rich-Ausgabe hinzuzufügen, kannst du die Methode...

after literally translation.

A german would not expect adverbs like "mühelos" (effortlessly) in a doc, because that sounds like advertisment.

@textualize Is it intended to keep that tone? If not I could work on a more appropriate translation.

**Comment by willmcgugan** [comment, marked as answer]
The README is somewhat of a marketing document. Although it shouldn't read too much like an ad.

When it comes to translations, I can only defer to someone who speaks the language. Feel free to open a PR.
